client/components/deposits-overview/recent-deposits.tsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { Flex, FlexItem, Icon } from '@wordpress/components';
+import { calendar } from '@wordpress/icons';
+
+/**
+ * Internal dependencies.
+ */
+import strings from './strings';
+import './style.scss';
+import DepositStatusChip from 'components/deposit-status-chip';
+import { getDepositDate } from 'deposits/utils';
+import { CachedDeposit } from 'wcpay/types/deposits';
+import { useDeposits } from 'wcpay/data';
+import { formatCurrency } from 'wcpay/utils/currency';
+import { Link } from '@woocommerce/components';
```

Comment:
Nit: this can be moved to "External dependencies"

Reply:
By this, I'm assuming you mean just the `Link` rather than the four lines